src/api/zoops.ts

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const zoopsRouter = express.Router();
+
+// Get /api/zoops/:id
+zoopsRouter.get("/:id", async (req, res, next) => {
+  try {
+    const zoopId = Number(req.params.id);
+    const zoop = await prisma.zoop.findFirst({
```

Comment:
I wouldn't use `findFirst` here - the user must be unique anyway. let's enforce that by using `findUnique`, or `findUniqueOrThrow`